src/lib/swapper/swappers/OsmosisSwapper/utils/helpers.ts

```diff
@@ -74,7 +74,7 @@ export const pollForComplete = (txid: string, baseUrl: string): Promise<string>
 export const getAtomChannelBalance = async (address: string, osmoUrl: string) => {
   const osmoResponseBalance = await (() => {
     try {
-      return axios.get(`${osmoUrl}/bank/balances/${address}`)
+      return axios.get(`${osmoUrl}/lcd/bank/balances/${address}`)
```

Comment:
Let's not tackle in this PR but not sure why we're calling the bank module from Cosmos SDK endpoint instead of using cosmos adapter's `getAccount()` here, looks hacky